src/test/java/com/fakng/fakngagrgtr/parser/apple/AppleParserTest.java

```diff
@@ -0,0 +1,110 @@
+package com.fakng.fakngagrgtr.parser.apple;
+
+import com.fakng.fakngagrgtr.configuration.ParserConfig;
+import com.fakng.fakngagrgtr.parser.AbstractParserTest;
+import com.fakng.fakngagrgtr.parser.LocationProcessor;
+import com.fakng.fakngagrgtr.persistent.company.Company;
+import com.fakng.fakngagrgtr.persistent.company.CompanyRepository;
+import com.fakng.fakngagrgtr.persistent.vacancy.Vacancy;
+import com.gargoylesoftware.htmlunit.WebClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class AppleParserTest extends AbstractParserTest {
+
+    private static final String URL = "Apple";
+    private AppleParser appleParser;
+    @Mock
+    private WebClient htmlWebClient;
+    @Mock
+    private CompanyRepository companyRepository;
+    @Mock
+    private LocationProcessor locationProcessor;
+
+    @BeforeEach
+    public void init() {
+        appleParser = new AppleParser(
+                htmlWebClient,
+                companyRepository,
+                locationProcessor,
+                URL,
+                new ParserConfig().objectMapper()
+        );
+    }
+
+    @Test
+    void getCompanyName() {
```

Comment:
Please, make it publick (it's a test) and update name to "testCompanyNameIsApple". Here domain is actually important, because it's the return value of the method